pallets/runtime/src/asset.rs

```diff
@@ -256,7 +272,7 @@ decl_storage! {
         pub Tokens get(fn token_details): map Ticker => SecurityToken<T::Balance>;
         /// Used to store the securityToken balance corresponds to ticker and Identity
         /// (ticker, DID) -> balance
```

Comment:
The comment could be updated. What happens if there are different identities in the key and in the value?

Reply:
That'll never happen unless there is a bug. We do not need to store did in the value at all for the backend. It is only there for the front end to easily populate a cap table since the did in the key is hashed and can not be recovered.